docs/src/tutorials/brief.md

```diff
@@ -8,7 +8,7 @@ In ImageFeatures.jl we have five methods to determine the vectors `X` and `Y` :
 - [`gaussian`](@ref) : `X` and `Y` are randomly sampled using a Gaussian distribution, meaning that locations that are closer to the center of the patch are preferred
 - [`gaussian_local`](@ref) : `X` and `Y` are randomly sampled using a Gaussian distribution where first `X` is sampled with a standard deviation of `0.04*S^2` and then the `Yi’s` are sampled using a Gaussian distribution – Each `Yi` is sampled with mean `Xi` and standard deviation of `0.01 * S^2`
 - [`random_coarse`](@ref) : `X` and `Y` are randomly sampled from discrete location of a coarse polar grid
-- [`centered`](@ref) : For each `i`, `Xi` is `(0, 0)` and `Yi` takes all possible values on a coarse polar grid
+- [`centre_sample`](@ref) : For each `i`, `Xi` is `(0, 0)` and `Yi` takes all possible values on a coarse polar grid
```

Comment:
Why "cent**re**_sample" and not "cent**er**_sample" ?

I ask because there is a function in `CoordinateTransformations` called `recenter` that we will likely use in `ImageTransformations`. Would be nice to have a consistent spelling.